app/commands/earthquake.rb

```diff
@@ -11,4 +11,7 @@
   update_timers_channel
   event.respond "Earthquake has been registered!"
   BOT.send_message(TIMER_ALERT_CHANNEL_ID, "**EARTHQUAKE**")
+  if defined? (EARTHQUAKE_ALERT_CHANNEL_ID) && EARTHQUAKE_ALERT_CHANNEL_ID.to_s.strip.length > 0
+    BOT.send_message(EARTHQUAKE_ALERT_CHANNEL_ID, "@raiders **EARTHQUAKE** **EARTHQUAKE** **EARTHQUAKE**")
```

Comment:
Thinking about this a little more, and trying to keep this more generic, what are your thoughts on also providing this message as an ENV variable?

Reply:
Hmm interesting idea. That actually would allow for customization of the message as well, i.e. when the Earthquake priority targets change week to week. Someone could go to update this ENV message to include the new targets. That might be some extra overhead though - is it possible for the discord bot itself to change the ENV (thinking that people would just want to interact with the discord bot, not have to go to heroku every week).

I could even see there actually being an extension to this, where people can type some sort of '!next-target Vox' command into the raid-chat channel, and this would get picked up by the bot which would send out an @raiders message to the batphone channel with that target. could save some time.


Anyway this is a bit of scope creep (i could see there being a separate bot from spawn timer bot tbh). 

The original problem that started this change was that, during this last earthquake on late saturday night, there were no officers on who could post a message to batphone with the word "earthquake", and some people missed it because of this (they weren't checking the normal raid-chat channel).

Reply:
I think you are over thinking it a bit. The issue I have with the static text is that it includes `@raiders` which may not be available in other servers. This bot is not a Castle specific bot, so I need to make sure that anything that that gets added is not specific. All we need to do is set an ENV in constants and use that for the message here.

Reply:
gotcha. Let's call it EARTHQUAKE_ALERT_MESSAGE. 

Does it need to have the same if defined and empty string checks as the channel ID? Should there be a default message of just "EARTHQUAKE" if EARTHQUAKE_ALERT_CHANNEL_ID is defined, but EARTHQUAKE_ALERT_MESSAGE is not?

Reply:
ok i added support for the EARTHQUAKE_ALERT_MESSAGE in env, and also a default message if channel is defined but the alert message is not. let me know if you'd rather i remove the comment as well..probably pretty obvious what the code is doing :)